src/defs.h

```diff
@@ -1469,6 +1469,7 @@ DECL_NETLINK(netfilter);
 DECL_NETLINK(route);
 DECL_NETLINK(selinux);
 DECL_NETLINK(sock_diag);
+DECL_NETLINK(generic);
```

Comment:
The name of the first argument sorts the sequence of DECL_NETLINK.
So putting `DECL_NETLINK(generic);` after `DECL_NETLINK(crypto);` may be better.

Reply:
Yep, will fix this.